packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1398,19 +1436,10 @@ export class MergeTree {
 		seq: number,
 		localSeq: number | undefined,
 		newSegments: T[],
-	) {
-		const continueFrom = (node: MergeBlock) => {
```

Comment:
Was this not used?

Reply:
Both this and onLeaf were moved outside of the mergeTree class due to the [consistent-function-scoping](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/consistent-function-scoping.md) rule.